android-lib/src/androidTest/java/com/kinvey/androidTest/store/user/MICLoginTest.java

```diff
@@ -62,7 +62,7 @@ public void testMIC_LoginWithAuthorizationCodeLoginPageWithClientId() throws Int
         DefaultKinveyMICCallback userCallback = loginWithAuthorizationCodeLoginPage(CLIENT_ID, REDIRECT_URI, client);
         assertNotNull(userCallback.myURLToRender);
         assertTrue(!userCallback.myURLToRender.isEmpty());
-        assertTrue(userCallback.myURLToRender.startsWith(client.getMICHostName() + client.getMICApiVersion() + "/oauth/auth?client_id=" + APP_KEY + "." + CLIENT_ID));
+        assertTrue(userCallback.myURLToRender.startsWith(client.getMICHostName() + client.getMicApiVersion() + "/oauth/auth?client_id=" + APP_KEY + "." + CLIENT_ID));
```

Comment:
please try to use `@JvmName("getMICApiVersion")` and `@JvmName("setMICApiVersion")` to keep the same names in Java

Reply:
Fixed.